specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/restorePoint.json

```diff
@@ -833,11 +830,13 @@
       "type": "object",
       "properties": {
         "status": {
-          "type": "object",
-          "items": {
-            "$ref": "./computeRPCommon.json#/definitions/InstanceViewStatus"
-          },
+          "$ref": "./computeRPCommon.json#/definitions/InstanceViewStatus",
           "description": "The resource status information."
+        },
+        "completionPercent": {
+          "type": "integer",
```

Comment:
Is there a way to represent this as nullable int or optional property?

Reply:
In the generated Sdk we can see this is generated as an nullable int. I had taken this from pre-exisiting diskSizeInGB which was also nullable

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/28188/files